build-tools/packages/build-cli/src/commands/bump.ts

```diff
@@ -109,6 +109,11 @@ export default class BumpCommand extends BaseCommand<typeof BumpCommand> {
 			this.error("ERROR: No dependency provided.");
 		}
 
+		// can be removed once server team owns their releases
+		if (args.package_or_release_group === MonoRepoKind.Server && flags.bumpType === "minor") {
+			this.error(`ERROR: Server release are always a ${chalk.bold("MAJOR")} release`);
```

Comment:
Regardless off the branch? Im guessing this is probably fine for now.

Reply:
This is fine when someone is just planning to do server minor release. The branch aspect comes into play when we want to consume server prereleases.